ironfish/src/network/peerNetwork.ts

```diff
@@ -677,6 +679,8 @@ export class PeerNetwork {
   private async onNewTransaction(
     message: IncomingPeerMessage<NewTransactionMessage>,
   ): Promise<boolean> {
+    const received = new Date(Date.now())
```

Comment:
A new date defaults to the current time

```suggestion
    const received = new Date()
```